docs/scripts/utils.py

```diff
@@ -32,9 +32,15 @@ def adjust_pictures(
 
     if line_index is not None:
         line = lines[line_index]
-        light_line = line.replace("reference external", "reference external only-light")
+        light_line = line
+        if "reference external" in line:  # for replacing the banner
```

Comment:
dont understand the addition of this if clause, what does it add? because if i remember correctly `string.replace('a','b')` will only do stuff if `a` is actually in `string`

Reply:
YOu are right, can probably remove, will verify.

Reply:
Changed this and it works, hence I'll resolve.